### DISCUSSION TITLE
Change receiver_id into receivertip

### DISCUSSION THREAD

**Discussion by vittore** [globaleaks/globaleaks-whistleblowing-software#4360] [Q&A] [answered]
Hi. 
I have a strange problem with version 4.18.8 and I ask the community for clarification.
We received a report but the recipient (report manager) did not remember the password and had lost the recovery key.
For this reason, we created a new user with the recipient role and assigned the channel on which the report had arrived.
When logging in with this second recipient, we see the report correctly in the list, but it is "disabled," that is, we cannot perform any operations.
Do you know how I can solve it?

I tried to change the receiver_id into receivertip table but so I got an error.

I really appreciate any help you can provide.
v.

**Comment by evilaliv3** [comment, marked as answer]
Dear @vittore 

Thank you for your request.

Editing the database manually is discouraged as you may incurr in data loss and there is no way possible that with such an operation you could enable an other user to access the same content.

If your system see enabled the escrow keys that enables administrators to change password of users, ask one administrator to reset the password of the user. This is unfortunately the only possibility to recover the data in case your recipient users have lost their password and their account recovery key.
